src/vma/dev/ring_bond.cpp

```diff
@@ -199,7 +199,9 @@ void ring_bond::restart()
 							}
 							for (j = 0; j < m_rx_flows.size(); j++) {
 								sockinfo* si = static_cast<sockinfo*> (m_rx_flows[j].sink);
-								p_ring_bond_netvsc->m_vf_ring->attach_flow(m_rx_flows[j].flow, m_rx_flows[j].sink);
+								if(!p_ring_bond_netvsc->m_vf_ring->attach_flow(m_rx_flows[j].flow, m_rx_flows[j].sink)) {
```

Comment:
Please fix style across the PR: add make sure you have spaces in the `if` construction.